ci-operator/step-registry/capz/test/aks-provision/capz-test-aks-provision-commands.sh

```
@@ -0,0 +1,69 @@
+echo "Creating resource group ${RESOURCEGROUP}"
+az group create --name "${RESOURCEGROUP}" --location "${AZURE_LOCATION}" --output none
+echo "${RESOURCEGROUP}" > "${SHARED_DIR}/resourcegroup_aks"
+
+K8S_VERSION_ARGS=()
+if [[ -n "${AKS_K8S_VERSION}" ]]; then
+  K8S_VERSION_ARGS=(--kubernetes-version "${AKS_K8S_VERSION}")
+else
+  LATEST_VERSION=$(az aks get-versions \
+    --location "${AZURE_LOCATION}" \
+    --output json \
+    --query 'max(orchestrators[?isPreview==`null`].orchestratorVersion)')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)